docs/source/install-django-project.rst

```diff
@@ -22,161 +22,120 @@ downloading data as a researcher.
 
-To reset the database:
 
-.. code-block:: shell
+Django
+~~~~~~
 
-   poetry run reset-db
+Here are few Django related tasks that might come up every now and then. 
 
-To reset the database and load an existing sql data file:
+To migrate the existing database:
 
 .. code-block:: shell
 
-   poetry run reset-db -s /location/of/sql/file
-
+   make migrate
 
-Create Superuser
-~~~~~~~~~~~~~~~~
-
-You can create a user through UI or if you'd rather have Superuser access you can create a user with the manage script.
-  
-Create a superuser by running:
+If you need to create an entry in the Site table:
 
 .. code-block:: shell
 
-   poetry run ./manage.py createsuperuser
-      
-Now you should be ready for anything. Going forward, you can run the server using the 
-directions below.
-
-Running the server
-~~~~~~~~~~~~~~~~~~~
-
-Again, this is covered above, but there is a case you'd need to run the development server with SSL.  This section will cover both variants. 
+   make site
 
-To run the Lookit server locally, run:
+To create a superuser:
 
 .. code-block:: shell
 
-   poetry run ./manage.py runserver
+   make superuser
 
-Or to use the invoke script:
+To run tests:
 
 .. code-block:: shell
 
-   poetry run invoke server
+   make tests
 
-Now you can go to http://localhost:8000 to see your local Lookit server! You should be able to log in using 
-the superuser credentials you created during setup.
 
-To view the HTTPS version of the local development add the ``https`` argument to the above command:
+Rabbitmq
+~~~~~~~~
+
+The broker should come up with the rest of the Docker services.  If you get a Celery worker error due to permissions, you can run the following command to resolve the issue and restart the worker service:
 
 .. code-block:: shell
 
-   poetry run invoke server --https
+   make broker-perms
 
-If you are not working extensively with lookit-api - i.e., if you just want to make some 
-new frames - you do not need to run celery, rabbitmq, or docker. For more information about 
-these services and how they interact, please see the `Contributing guidelines <https://github.com/lookit/lookit-api/blob/develop/CONTRIBUTING.md>`__.
+Postgres
+~~~~~~~~
 
-Celery 
-~~~~~~~~~~~~~~
+Here are a couple of command that might be useful for managing the local database.
 
-You should already have a rabbitmq server installed and running.  You can check this by:
+To access the database shell:
 
 .. code-block:: shell
 
-   docker ps -f name=lookit-rabbit
-   
-If rabbitmq is not running, you can start it using:
-
-.. code-block:: shell
+   make dbshell
 
-   docker start lookit-rabbit
-
-Then use the invoke command to start the celery worker:
+To import a SQL file into a fresh database (one where migrations haven't been ran):
 
 .. code-block:: shell
 
-   poetry run invoke celery-service
+   cat /location/of/sql/file | make dbpipe
+
 
 Authentication
```

Comment:
This section says "you will need to log in using the superuser you created earlier using manage.py", which I think is referencing `make superuser` on line 104, but I don't see any references to "manage.py" on this page, so that might be confusing. Is that left over from an earlier version?